core/services/offchainreporting/discoverer_database.go

```diff
@@ -0,0 +1,62 @@
+package offchainreporting
+
+import (
+	"context"
+	"database/sql"
+
+	"github.com/lib/pq"
+	p2ppeer "github.com/libp2p/go-libp2p-core/peer"
+	"github.com/pkg/errors"
+	ocrnetworking "github.com/smartcontractkit/libocr/networking"
+	"go.uber.org/multierr"
+)
+
+var _ ocrnetworking.DiscovererDatabase = &DiscovererDatabase{}
+
+type DiscovererDatabase struct {
```

Comment:
this breaks the existing pattern of just adding fields to the existing DB object in database.go and just using interfaces to limit scope (like what OCRContractTrackerDB does) why not do the same for this? Also separately we should rename database.go to orm.go to keep it consistent with the other service packages

Reply:
It breaks the pattern because the ORM is scoped to a particular OCR job - whereas this is scoped to a particular peer.

Reply:
It's called `database.go` because that is what it's called internally in OCR